Mobile Test Runner for Jama/Mobile Test Runner for Jama/TestListViewController.swift

```diff
@@ -97,14 +95,15 @@ extension TestListViewController: EndpointDelegate {
                 
                 case .cycle:
                     let tmpList = TestCycleListModel()
-                    tmpList.extractCycleList(fromData: unwrappedData)
+                    tmpList.extractCycleList(fromData: unwrappedData, parentId: self.planId)
                     if tmpList.testCycleList.isEmpty {
                         return
                     }
                     self.testCycleList.testCycleList.append(contentsOf: tmpList.testCycleList)
-
+                    self.totalCyclesVisable = self.testCycleList.testCycleList.count
+                    self.testPlanList.testPlanList[self.selectedPlanIndex].numOfCycles = self.totalCyclesVisable
                     //reload Data in view? self.testCycle.reloadData()
```

Comment:
what is this comment signaling? Is this work that needs to be done or is it out-of-date?